### PR TITLE
Update the ML tutorial

### PR DESCRIPTION
**Pull request by tbedford**
## Description

* Updates the machine learning tutorial.
* Applies new structure.
* Updates screenshots.
* Updates example code to use Quix Streams.
* Tests all steps.

## Review

* [Page to review](https://quixdocsdev.blob.core.windows.net/pr143/platform/tutorials/train-and-deploy-ml/index.html)

### CODE REVIEW

File: docs/platform/tutorials/train-and-deploy-ml/images/enable-topic-persistence.png
Comment (SteveRosam) [inline docs/platform/tutorials/train-and-deploy-ml/images/enable-topic-persistence.png:1]: i think people might get confused with this topic name. in the text it talks about the F1 data so maybe the topic in the screenshot should just be "f1-data"?

Comment (SteveRosam) [reply docs/platform/tutorials/train-and-deploy-ml/images/enable-topic-persistence.png:1]: and. IMO the green line looks too much like part of the UI.

Comment (SteveRosam) [reply docs/platform/tutorials/train-and-deploy-ml/images/enable-topic-persistence.png:1]: just put it at an angle like the rest.

**Comment by SteveRosam** [comment]
@tbedford minor comment

**Review by SteveRosam — CHANGES_REQUESTED** [review]

File: docs/platform/tutorials/train-and-deploy-ml/images/enable-topic-persistence.png
Comment (tbedford) [reply docs/platform/tutorials/train-and-deploy-ml/images/enable-topic-persistence.png:1]: I've updated the screenshot. Hopefully it's clearer.

File: docs/platform/tutorials/train-and-deploy-ml/train-ml.md [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+
+## Install the required libraries
+
+You now need to install some Python libraries on your system. This is the system where you are running Jupyter Notebook. If you do not have Jupyter Notebook installed, please refer to the [prerequisites](./index.md#prerequisites) for this tutorial.
+
+The following libraries are required to create the ML model and visualize your data. To install the libraries enter the following commands into a terminal on the system where you are running Jupyter Notebook:
+
+``` shell
+python3 -m pip install seaborn
+python3 -m pip install sklearn
```
Comment (SteveRosam) [inline docs/platform/tutorials/train-and-deploy-ml/train-ml.md]: now called `scikit-learn` i think

File: docs/platform/tutorials/train-and-deploy-ml/train-ml.md [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+This is just one approach that you might use if you are already familiar with Jupyter Notebook. You might also train your ML model directly in Quix using live data, or data played back using the [replay feature](../../how-to/replay.md).
+
+## Install the required libraries
+
+You now need to install some Python libraries on your system. This is the system where you are running Jupyter Notebook. If you do not have Jupyter Notebook installed, please refer to the [prerequisites](./index.md#prerequisites) for this tutorial.
+
+The following libraries are required to create the ML model and visualize your data. To install the libraries enter the following commands into a terminal on the system where you are running Jupyter Notebook:
+
+``` shell
+python3 -m pip install seaborn
```
Comment (SteveRosam) [inline docs/platform/tutorials/train-and-deploy-ml/train-ml.md]: I need to remove `python3 -m` for mine to work. Im on windows

File: docs/platform/tutorials/train-and-deploy-ml/deploy-ml.md [hunk 3]
```diff
@@ -1,108 +1,80 @@
...
+6. Change the name to `Prediction Model`.
 
-Now let's run the model you created in the previous article. If you have your own model and already know how to run the Python to execute it then these steps might also be useful for you.
+7. Ensure the input is `f1-data`.
 
-Ensure you are logged into the Quix Portal
+8. Leave output as `hard-braking` (its default value).
 
-1. Navigate to the `Code Samples`
+9. Click `new topic` next to `hard-braking` to create this topic, leaving topic values at their defaults.
```
Comment (SteveRosam) [inline docs/platform/tutorials/train-and-deploy-ml/deploy-ml.md]: This step isn't necessary

File: docs/platform/tutorials/train-and-deploy-ml/deploy-ml.md [hunk 4]
```diff
@@ -1,108 +1,80 @@
...
-
-4. Now click Edit code
-
-5. Change the name to "Prediction Model"
-
-6. Ensure the input is "f1-data"
+		You can see a list of projects at any time by clicking `Projects` in the left-hand navigation.
 
-7. Ensure the output is "brake-prediction"
+11. Click `Topics` in the left-hand navigation. Wait until the topic `hard-braking` has been created.
```
Comment (SteveRosam) [inline docs/platform/tutorials/train-and-deploy-ml/deploy-ml.md]: Is there supposed to be a step after this? If not it's probably not needed.

File: mkdocs.yml [hunk 5]
```diff
@@ -55,9 +55,12 @@ nav:
...
-      - 'Train and deploy an ML model':
-        - 'How to train an ML model': 'platform/tutorials/train-and-deploy-ml/train-ml-model.md'
-        - 'Run ML model in realtime environment': 'platform/tutorials/train-and-deploy-ml/deploy-ml.md'
+      - 'Real-time Machine Learning (ML) pipelines':
+        - platform/tutorials/train-and-deploy-ml/index.md
+        - 'Create your data': 'platform/tutorials/train-and-deploy-ml/create-data.md'
+        - 'Import data into Jupyter': 'platform/tutorials/train-and-deploy-ml/import-data.md'
+        - 'Train your ML model': 'platform/tutorials/train-and-deploy-ml/train-ml.md'
+        - 'Deploy your ML model': 'platform/tutorials/train-and-deploy-ml/deploy-ml.md'
       - 'Data Science': 'platform/tutorials/data-science/data-science.md'
```
Comment (SteveRosam) [inline mkdocs.yml]: link has changed

Comment (SteveRosam) [reply mkdocs.yml]: index.md i think

**Review by SteveRosam — CHANGES_REQUESTED** [review]
Minor changes needed

File: docs/platform/tutorials/train-and-deploy-ml/conclusion.md
Comment (SteveRosam) [inline docs/platform/tutorials/train-and-deploy-ml/conclusion.md:1]: Add conclusion to nav?

**Review by SteveRosam — CHANGES_REQUESTED** [review]

File: docs/platform/tutorials/train-and-deploy-ml/train-ml.md (hunk 1 above)
Comment (tbedford) [reply docs/platform/tutorials/train-and-deploy-ml/train-ml.md]: Yes, fixed.

File: docs/platform/tutorials/train-and-deploy-ml/train-ml.md (hunk 2 above)
Comment (tbedford) [reply docs/platform/tutorials/train-and-deploy-ml/train-ml.md]: Removed. Just pip commands used now.

File: docs/platform/tutorials/train-and-deploy-ml/deploy-ml.md (hunk 3 above)
Comment (tbedford) [reply docs/platform/tutorials/train-and-deploy-ml/deploy-ml.md]: Removed.

File: docs/platform/tutorials/train-and-deploy-ml/deploy-ml.md (hunk 4 above)
Comment (tbedford) [reply docs/platform/tutorials/train-and-deploy-ml/deploy-ml.md]: It was just so they could see it being created. Removed.

File: mkdocs.yml (hunk 5 above)
Comment (tbedford) [reply mkdocs.yml]: You mean the link for the data science tutorial?

File: docs/platform/tutorials/train-and-deploy-ml/conclusion.md
Comment (tbedford) [reply docs/platform/tutorials/train-and-deploy-ml/conclusion.md:1]: Good catch. Fixed.

**Review by SteveRosam — APPROVED** [review]

**Pull request merged**